rpc/handlers.go

```diff
@@ -392,6 +395,11 @@ func (h *Handler) MethodsV0_6() ([]jsonrpc.Method, string) { //nolint: funlen
 			Params:  []jsonrpc.Parameter{{Name: "contract_address"}, {Name: "key"}, {Name: "block_id"}},
 			Handler: h.StorageAt,
 		},
+		{
+			Name:    "starknet_getStorageProof",
```

Comment:
Should we also support this in `Methods()` (for rpcv7)? I guess it should techincally only be for rpcv8, but we don't have a function for that atm..